### ISSUE TITLE
Fails to run with minimal config

### ISSUE THREAD

**Issue by tsudmi** [prysmaticlabs/prysm#4623]
Hi,

I'm running Validators and Beacon node with `--minimal-config` and validators start getting penalties as soon as they are activated:
```
time="2020-01-23 19:59:06" level=error msg="Could not get validator index in assignment: index 0 not in committee" prefix=validator
time="2020-01-23 19:59:12" level=info msg="New Balance" epoch=2348 newBalance=31.990002145 percentChange="-0.00323%" prefix=validator prevBalance=31.991034136 pubKey=0xb1a061e0aeeafde4
```
However, Validators in genesis are not getting any penalties.

Beacon chain and validator client versions: `HEAD-e96c2f`

**Comment by tsudmi** [comment]
@prestonvanloon , Could it be somehow related to https://github.com/prysmaticlabs/prysm/pull/4566 ?

**Comment by tsudmi** [comment]
Looks like only the last added deposit is affected by this issue. All the previous deposits are not getting any penalties.

**Comment by tsudmi** [comment]
Looks like `activation_epoch` could be behind `deposit_inclusion_slot`. For example, deposit_inclusion_slot=45031, activation_epoch=5594 with 8 slots per epoch.

**Comment by 0xKiwi** [comment]
How are you using `--minimal-config`? To sync to the testnet?

**Comment by tsudmi** [comment]
Hi @0xKiwi , thanks for the response.

I've set up a private network, where both validators and beacon nodes are running with `--minimal-config`, so it's independent of the Prysm testnet.

**Comment by 0xKiwi** [comment]
Okay great, @tsudmi

How did you start the clients? Through bazel or docker?

**Comment by tsudmi** [comment]
I tried both, the error is the same.

**Comment by 0xKiwi** [comment]
Got it, if you're running through bazel (not sure about docker atm), you'll need the `--define ss=minimal` flag before you start defining flags for the client (before the `--`).

Did you use that flag along with the `--minimal-config` flag?

**Comment by tsudmi** [comment]
I thought to try running with that flag, it's just that the whole setup takes quite long to prepare (have to wait for 1024 blocks for ETH1 chain). However, it helped me with a similar issue (#4469).

It would be great to have a similar flag with docker images.

**Comment by rauljordan** [comment]
Hey @tsudmi you can change the eth1 follow distance in our config parameters under `shared/params/config.go` to 16 instead of 1024 and that will help so you don't have to wait too long for the eth1 chain. Can we close this?

**Comment by tsudmi** [comment]
Hi @rauljordan ,
Looks like it should be `16` according to the [minimal config](https://github.com/ethereum/eth2.0-specs/blob/dev/configs/minimal.yaml#L34) in eth2.0-specs. I've created a PR on that (https://github.com/prysmaticlabs/prysm/pull/4566), but it's not merged yet due to failing tests.

**Comment by tsudmi** [comment]
Fixed in #4566
